.travis.yml

```diff
@@ -49,18 +49,27 @@ jobs:
 
     - env: &py36_env
       - PYTHON=3.6
+      - NUMPY=1.14.0
+      - PANDAS=0.22.0
```

Comment:
Test failure due to the hdfs tests PR being merged (sorry about that). You need to add `- *test_and_lint` to this env, the other builds look to be gotten by the merge. To be safe you may try rebasing on master to ensure all builds include that line after merging.

Reply:
No worries. Thanks for the pointer. Have merged with `master` and added that line.